plugins/package-managers/cocoapods/src/main/kotlin/PodDependencyHandler.kt

```diff
@@ -111,6 +115,43 @@ internal class PodDependencyHandler : DependencyHandler<Lockfile.Pod> {
         }
     }
 
+    private fun File.parsePodspecFile(): Podspec? {
+        val content = readText()
+
+        return if ("Pod::Spec.new" in content) {
```

Comment:
Please extract the content of this if black to a separate function for readability.